src/allEqual.js

```diff
@@ -0,0 +1,23 @@
+import { uniq, length, lte, __, pipe } from 'ramda';
+
+/* eslint-disable */
+/**
+  * Checks that all elemnets in a list are the same
```

Comment:
Returns true if all elements of the list match the each other, false if there are any that don't.